skimage/_shared/utils.py

```diff
@@ -415,3 +415,21 @@ def _validate_interpolation_order(image_dtype, order):
              FutureWarning, stacklevel=2)
 
     return order
+
+
+def _to_np_mode(mode):
```

Comment:
I suggested this changes for an other PR, but it is fine by me here :slightly_smiling_face:

Reply:
This was also added in #5173, so I will probably have to rebase whichever one is merged second.